--- ./arch/arm/boot/dts/suniv-f1c100s.dtsi	2022-02-23 12:26:21.759741281 +0800
+++ ./arch/arm/boot/dts/suniv-f1c100s.dtsi	2022-02-23 12:26:21.759741281 +0800
@@ -209,7 +209,7 @@
 
 			lcd_rgb666_pins: lcd-rgb666-pins {
 				pins = "PD0", "PD1", "PD2", "PD3", "PD4",
-				       "PD5", "PD6", "PD7", "PD8", "PD9",
+				       "PD5", "PD6", "PD8", "PD9",
 				       "PD10", "PD11", "PD12", "PD13", "PD14",
 				       "PD15", "PD16", "PD17", "PD18", "PD19",
 				       "PD20", "PD21";
